PowerEditor/src/ScintillaComponent/FindReplaceDlg.h

```diff
@@ -568,13 +568,8 @@ class Progress
 		return false;
 	}
 
-	void setInfo(const TCHAR *info) const
-	{
-		if (_hwnd)
-			::SendMessage(_hPText, WM_SETTEXT, 0, reinterpret_cast<LPARAM>(info));
-	}
-
-	void setPercent(unsigned percent, const TCHAR *fileName) const;
+	void updateSearchProgress(unsigned percent, const TCHAR* fileName, int nbHitsSoFar) const;
+	void updateSearchProgressDuringCancel(const TCHAR* info) const;

```

Comment:
the function `updateSearchProgressDuringCancel` makes confusing.
Please restore `setInfo` and `setPercent`.